presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -218,9 +223,8 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateErrorTask(
       : "1970-01-01";
 
   std::stringstream ss;
-  ss << baseSpillPath << "/";
-  ss << dateString;
-  ss << "/";
+  ss << baseSpillPath << "/" << nodeIp << "/" << nodeId << "/" << dateString
```

Comment:
Maybe we could combine nodeIp and nodeId in one layer to reduce the depth of the directory tree, consider in form of <nodeId_nodeIp>? Thanks!